lib/watts.rb

```diff
@@ -236,6 +236,8 @@ def self.inherited base
 					end
 
 					# TODO:  Problems.
+					content_type=response.header['Content-Type'] || 'text/html;charset=utf8' 
```

Comment:
The main issue with this change is that it creates a `Rack::Response` even if it doesn't exist.  I was thinking about this, it's come up a couple of times.  I think what would probably be best to do is add something like `Watts::Resource#default_content_type` or possibly `Watts::Resource#default_headers` so that it can be overridden in subclasses.